src/universal/utils/meetings/helpLookups.js

```diff
@@ -15,36 +15,36 @@ import {
   LAST_CALL
 } from 'universal/utils/constants'
 
-const makeLink = (href, copy) => (
-  <a href={href} rel='noopener noreferrer' target='blank' title={copy}>
-    {copy}
-  </a>
-)
+const makeLink = (link, copy) => {
+  const href = `https://www.parabol.co/getting-started-guide/${link}`
+  return (
+    <a href={href} rel='noopener noreferrer' target='blank' title={copy}>
+      {copy}
+    </a>
+  )
+}
 
-const teamAgendaHelpLink = makeLink(
-  'https://www.parabol.co/getting-started-guide#team-agenda',
-  'Learn More'
-)
+const teamAgendaHelpLink = makeLink('action-meetings-101#team-agenda', 'Learn More')
 
-const actionGettingStartedLink = makeLink(
-  'https://www.parabol.co/getting-started-guide',
-  'Getting Started Guide'
-)
-// TODO: Add Learn More links, etc. for Retro Guide when it is created
-const retroGettingStartedLink = makeLink(
-  'https://www.parabol.co/getting-started-guide',
-  'Getting Started Guide'
-)
+const actionGettingStartedLink = makeLink('action-meetings-101', 'Getting Started Guide')
+
+const retroGettingStartedLink = makeLink('retrospective-meetings-101', 'Getting Started Guide')
 
-// TODO: Refactor to handle Retro type
-const meetingLabel = 'an Action Meeting' || 'a Retro Meeting'
-const meetingHelpLink = actionGettingStartedLink || retroGettingStartedLink
+const retroGettingStartedContent = (
+  <p>
+    {'See our '}
+    {retroGettingStartedLink}
+    {' for running a Retrospective Meeting.'}
+  </p>
+)
+const checkInFacilitatorBarTip =
+  'Facilitator: allow each teammate a moment to answer today’s prompt, then mark them as Here or Not Here.'
 
-const lobbyHelpContent = (
+const actionLobbyHelpContent = (
```

Comment:
this file is starting to get a little large. let's break things like this into their own separate component. Then we can async load them when the user clicks on em.